### ISSUE TITLE
Icon requests

### ISSUE THREAD

**Issue by Donkey-Doug** [dkanada/frost#490]
- Banking (ing)
- Aves
- Quick Bible
- Barinsta
- Showly 2.0
- Trace
- DeepL
- Fritter
- PDF viewer plus
- Stock widget
- The latest AOSP calculator icon

File: 
[20210704_031523.zip](https://github.com/dkanada/frost/files/6759236/20210704_031523.zip)

All apps are open source, except the banking app.

**Comment by Donkey-Doug** [comment]
I think pdf viewer plus is already in your icon pack, but it does not recognize it. Probably because there exists a play store and fdroid version.

**Comment by Donnnno** [comment]
An icon for fritter is already created and ready to be merged here: #487

**Comment by baitmooth** [comment]
-- | -- | -- | -- | --
-- | -- | -- | -- | --
aves (*n*)| banking (*n*)| deepl (*n*)| quickbible (*n*)| showly20 (*n*)
stockswidget (*n*)|
*total icons*: **6**

![490-2021-07-25-1226](https://user-images.githubusercontent.com/23211478/126899360-a0139bb0-d95c-4112-a955-f9a510afeee7.png)

**Comment by Donkey-Doug** [comment]
Wow, that was fast! Thank you.

**Comment by Donkey-Doug** [comment]
Barinsta also already exists.

https://github.com/dkanada/frost/issues/443

**Comment by baitmooth** [comment]
> Barinsta also already exists.
> 
> #443

Yes, I know, I drew it. But i have some strange workflow with branches per issue, which leads to not showing icons done elsewhere.
But I could have mentioned it...

**Comment by baitmooth** [comment]
I could also somehow mention that I decided against drawing some kinds of icons, but I haven't found the right place for that.
The lion for this banking app is tempting, but its CSS and banking. Also I don't draw cryptocurrency apps or CSS games with ads or IGP. It was too depressing.
  ¯\_(ツ)_/¯

**Comment by baitmooth** [comment]
> Barinsta also already exists.
> 
> #443

And sometimes I make a checklist in my comments to make it easier for other to track what is already done. But I forgot it this time. SRY for that.

**Comment by baitmooth** [comment]
could not resist this lion...

**Comment by Donkey-Doug** [comment]
The lion is awesome. Thank you.
